llvm/lib/IR/Core.cpp

```diff
@@ -851,13 +851,7 @@ LLVMTypeRef LLVMPointerType(LLVMTypeRef ElementType, unsigned AddressSpace) {
   return wrap(PointerType::get(unwrap(ElementType), AddressSpace));
 }
 
-LLVMBool LLVMPointerTypeIsOpaque(LLVMTypeRef Ty) {
-#ifdef INTEL_SYCL_OPAQUEPOINTER_READY
-  return true;
-#else // INTEL_SYCL_OPAQUEPOINTER_READY
-  return unwrap(Ty)->isOpaquePointerTy();
-#endif // INTEL_SYCL_OPAQUEPOINTER_READY
-}
+LLVMBool LLVMPointerTypeIsOpaque(LLVMTypeRef Ty) { return true; }
```

Comment:
I think we can remove this function and its uses? But that needs to happen in llorg though.

Thanks

Reply:
Yes, we will sync with llorg on this. We will start to remove usage of it in our own code first, after removing code guarded in INTEL_SYCL_OPAQUEPOINTER_READY. After that, we will consider removing it if community also remove it.